src/main_frontend.cpp

```diff
@@ -888,6 +902,8 @@ ROM management options:
     FE_PrintAudioDevices();
 }
 
+#undef main
```

Comment:
`#undef main` is unreliable
It gave me undefined reference error to `SDL_main`
Try `#define SDL_main` which worked for me.